### PR TITLE
Adds announcement function.

### LINKED ISSUE #12: Announcement Feature
A command where you can announce to a particular channel given the channel id and the message.

Basically, receive the message from the user, receive the channel id as argument, send that particular message to that channel.

### PR DESCRIPTION
**Pull request by yagueto**
Adds an announcement function which sends a message given a channel, called by 
```markdown
 (prefix)announce #channel message to announce
```

Fixes IEEE-VIT/heimdall#12

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -143,6 +144,19 @@ async def invites(ctx, *args):
         await ctx.send(link)
 
 
+@bot.command()
+async def announce(ctx, channel:discord.TextChannel, *args):
+    await channel.send(f"📢 " + " ".join(str(x) for x in args) + " 📢")
```
Comment (mintbomb27) [inline bot.py:149]: Please remove the 📢s in between the message. As the user could add them if needed.

Comment (mintbomb27) [reply bot.py:149]: And the problem with this approach is that, if the user sends a properly formatted message with tabs and spaces, the message will be just joined with whitespaces. 

So if i send a message like this, 
Screenshot 2021-10-03 at 11 54 59 AM

The output will be this:
Screenshot 2021-10-03 at 11 55 51 AM

So figure out a method to even take in whitespaces and tabs and all!

**Review by mintbomb27 — CHANGES_REQUESTED** [review]
Please see the comments. Thank you for your contribution!

File: bot.py (hunk 1 above)
Comment (yagueto) [reply bot.py]: The new commit solves the issue

Comment (mintbomb27) [reply bot.py]: Yes it does!

**Review by mintbomb27 — APPROVED** [review]
LGTM! Great work. Thanks a lot for the contribution! Do checkout our other repos at our [organisation](https://github.com/IEEE-VIT/).

**Comment by yagueto** [comment]
Thanks!

**Pull request merged**